### PR TITLE
internet plugins for using of Korean bank

### PR DESCRIPTION
**Pull request by daehyunpy**
add cask files for using of Korean bank: aod-firewall.rb, cross-web.rb, ip-inside-agent-citi.rb, ip-inside-agent-hanai.rb, veraport.rb, xecure-web-unified.rb
checked bank list: Hana, Shinhan, Citi, NH

### CODE REVIEW

File: Casks/aos-firewall.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
+cask :v1 => "aos-firewall" do
+  version :latest
+  sha256 :no_check
+
+  url "http://ahnlabdownload.nefficient.co.kr/aos/sup/aosfirewall.dmg"
```
Comment (vitorgalvao) [inline Casks/aos-firewall.rb]: Double quotes are reserved for cases where string interpolation is needed. Please change these to single quotes. Same for every other stanza in all casks of this PR.

File: Casks/aos-firewall.rb [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  name "AOS Firewall"
+  homepage "http://www.ahnlab.com"
+  license :gratis
+
+  pkg "AOSFirewall.pkg"
+  uninstall :pkgutil =>
+    [
+        # "com.ahnlab.firewallStatus",
```
Comment (vitorgalvao) [inline Casks/aos-firewall.rb]: Remove the comment.

File: Casks/cross-web.rb [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+  version :latest
+  sha256 :no_check
+
+  url "https://open.citibank.co.kr/3rdParty/initech/plugin/down/npCrossWeb_Mac.pkg"
+  name "Cross WEB"
+  homepage "https://open.citibank.co.kr"
+  license :gratis
+
+  pkg "npCrossWeb_Mac.pkg"
+  uninstall :script => "/Applications/CrossWeb/UninstallCrossWeb.app/Contents/MacOS/UninstallCrossWeb"
```
Comment (vitorgalvao) [inline Casks/cross-web.rb]: This isn’t a script (neither are any any of the others). This simply calls a GUI, so we can’t accept this as is.

There are [scripts](https://github.com/caskroom/homebrew-cask/tree/master/developer/bin) and [documentation](https://github.com/caskroom/homebrew-cask/blob/master/doc/CASK_LANGUAGE_REFERENCE.md#uninstall-stanza-details) on the right way to do it. You should be looking at `:pkgutil`.

File: Casks/aos-firewall.rb [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+  version :latest
+  sha256 :no_check
+
+  url "http://ahnlabdownload.nefficient.co.kr/aos/sup/aosfirewall.dmg"
+  name "AOS Firewall"
+  homepage "http://www.ahnlab.com"
+  license :gratis
+
+  pkg "AOSFirewall.pkg"
+  uninstall :pkgutil =>
```
Comment (vitorgalvao) [inline Casks/aos-firewall.rb]: [Example](https://github.com/caskroom/homebrew-cask/blob/7fb617c0b96ebdaa932f76c118d518b4ba193c7a/Casks/teamviewer.rb) of how this should be structured.

File: Casks/xecure-web-unified.rb [hunk 5]
```diff
@@ -0,0 +1,12 @@
+cask :v1 => "xecure-web-unified" do
```
Comment (vitorgalvao) [inline Casks/xecure-web-unified.rb]: Should be (and the file name) `xecureweb-unified`.

File: Casks/ip-inside-agent-hana.rb [hunk 6]
```diff
@@ -0,0 +1,12 @@
+cask :v1 => "ip-inside-agent-hana" do
```
Comment (vitorgalvao) [inline Casks/ip-inside-agent-hana.rb]: `ip-inside-agent-for-hana-bank`.

File: Casks/ip-inside-agent-citi.rb [hunk 7]
```diff
@@ -0,0 +1,12 @@
+cask :v1 => "ip-inside-agent-citi" do
```
Comment (vitorgalvao) [inline Casks/ip-inside-agent-citi.rb]: `ip-inside-agent-for-citi-bank`.

Comment (jawshooah) [inline Casks/ip-inside-agent-citi.rb]: Why not just use `citibank-ip-inside-agent`, with the name changed to match? (Citibank is one word, not two).

Comment (vitorgalvao) [inline Casks/ip-inside-agent-citi.rb]: Sure, that works too. As long as they match, no objections.

**Comment by hyuna917** [comment]
thank you for all. let me fix it ><

File: Casks/ip-inside-agent-citi.rb (hunk 7 above)
Comment (hyuna917) [inline Casks/ip-inside-agent-citi.rb]: thank you for advice. i think i wanna use these names: `ip-inside-agent-for-citi-bank` `ip-inside-agent-for-hana-bank`

Comment (vitorgalvao) [inline Casks/ip-inside-agent-citi.rb]: `citibank` at least should be a single word, though, as @jawshooah mentioned.

File: Casks/cross-web.rb (hunk 3 above)
Comment (hyuna917) [inline Casks/cross-web.rb]: i tried to use ‘`pkgutil` but i couldn’t find identifier of plugins. how can i find it?

Comment (vitorgalvao) [inline Casks/cross-web.rb]: See the documentation I linked to. If you use [the first command](https://github.com/caskroom/homebrew-cask/blob/master/doc/CASK_LANGUAGE_REFERENCE.md#uninstall-key-pkgutil) you’ll see that for this cask, for example, `:pkgutil`’s value is `kr.co.iniline.pkg.CrossWebPackage`.

**Pull request closed without merging**